src/plugins/discover/public/application/angular/get_inner_angular.ts

```diff
@@ -16,7 +16,7 @@ import 'angular-sanitize';
 import { EuiIcon } from '@elastic/eui';
 import { i18nDirective, i18nFilter, I18nProvider } from '@kbn/i18n/angular';
 import { CoreStart, PluginInitializerContext } from 'kibana/public';
-import { DataPublicPluginStart } from '../../../../data/public';
+import { DataPublicPluginStart } from 'src/plugins/data/public';
```

Comment:
🧹 don't think it's needed to change this, guess it's a bug-hunt-leftover

Reply:
does it hurt? I thought we preferred absolute as opposed to relative paths?

Reply:
I don't feel hurt :)